wealth-management-client/.angular-cli.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
```

Comment:
Thanks for the image... it is looking good.
Good start 👍 
Need some changes in the data will do it...